preprocess.py

```diff
@@ -1,48 +1,53 @@
 import re
 
+
 def read_text(filename: str) -> str:
     """
         This function reads a file given a file path and converts contents into a string. 
     """
-    with  open(filename,'r',encoding='utf-8',newline='\n') as file:
-        
+    with  open(filename, 'r', encoding='utf-8', newline='\n') as file:
         file_string = file.read()
 
         return file_string
 
+
 # Helper functions for preproc_text()
 def __remove_new_line(string: str) -> str:
     """
     Return a copy of the string with new lines replaced with a single space.
     """
     return string.replace("\n", " ")
 
+
 def __remove_contractions(string: str) -> str:
     """
     Return a copy of the string with English contractions replaced.
     WARNING: will change word count.
     """
-    string = re.sub(r'\'s', ' is', string) 
```

Comment:
thank you for fixing the spacing!

Reply:
Of course, ideally we will have a standardized linting process for the project :)